app.js

```diff
@@ -0,0 +1,122 @@
+// Book Class: Represents a Book
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+// Store Class: Handles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(id) {
+    const books = Store.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.id === id) {
+        books.splice(index, 1);
```

Comment:
* This is an OPTIONAL suggestion as it is not a requirement for this project but i think in your spare time you can work on this.
* Using the array method `splice` might be expensive as it mutates the original array which may lead to unexpected and hard-to-debug issues [Read more on why it is not advisable](https://blog.sapegin.me/all/avoid-mutation/)